examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,58 +14,56 @@
   },
```

Comment:
<p>This is not very informative, at least to me, so I'd use <code>plot_posterior</code>  like we used above. It should automatically create two plots, one per step function an as we are customizing the dims and coords, it will be correctly labeled automatically</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>